README.md

```diff
@@ -1,86 +1,129 @@
 
-## .NET Framework 4.6.2
+[![Downloads from Docker Hub](https://img.shields.io/docker/pulls/microsoft/dotnet-framework.svg)](https://hub.docker.com/r/microsoft/dotnet-framework)
+[![Stars on Docker Hub](https://img.shields.io/docker/stars/microsoft/dotnet-framework.svg)](https://hub.docker.com/r/microsoft/dotnet-framework)
 
-The .NET Framework 4.6.2 is included in the [Windows Server Core base image](https://hub.docker.com/r/microsoft/windowsservercore/). You can pull/FROM microsoft/windowsservercore to use .NET Framework 4.6.2.
+# What is the .NET Framework?
 
-To make discovery easier, we have also provided the same image (with a FROM wrapper) in this repo, enabling you to pull `microsoft/dotnet-framework`.
+The [.NET Framework](https://www.microsoft.com/net/framework) is a general purpose development platform maintained by Microsoft. It is the most popular way to build client and server applications for Windows and Windows Server. It is included with Windows, Windows Server and Windows Server Core. It includes server technologies such as ASP.NET Web Forms, ASP.NET MVC and Windows Communication Framework (WCF) applications, which you can run in Docker containers.
 
-- Optional tags: `latest`, `4.6.2`
-- Dockerfile: [4.6.2/Dockerfile](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.6.2/Dockerfile)
+.NET has several capabilities that make development easier, including automatic memory management, (runtime) generic types, reflection, asynchrony, concurrency, and native interop. Millions of developers take advantage of these capabilities to efficiently build high-quality web and client applications.
 
-## .NET Framework 3.5
+You can use C#, F# and VB to write .NET Framework apps. C# is simple, powerful, type-safe, and object-oriented while retaining the expressiveness and elegance of C-style languages. F# is a multi-paradigm programming language, enabling both functional and object-oriented patterns and practices. VB is a rapid development programming language with the deepest integration between the language and Visual Studio, providing the fastest path to a working app.   
 
-.NET Framework 3.5 is an optional component in Windows Server Core. The official Docker image for the .NET Framework 3.5 is included in this Docker repository and is in preview. You can pull/FROM microsoft/dotnet-framework:3.5 to use .NET Framework 3.5.
+The .NET Framework was first released by Microsoft in 2001. The latest version is [.NET Framework 4.6.2](https://www.microsoft.com/net/framework).
 
-- Required tag: `3.5`
-- Dockerfile: [3.5/Dockerfile](https://github.com/Microsoft/dotnet-framework-docker/blob/master/3.5/Dockerfile)
+> https://docs.microsoft.com/dotnet/articles/framework/
 
-## .NET Framework 4.6 and 3.5 and ASP.NET
+![dotnet-icon](https://cloud.githubusercontent.com/assets/2608468/19943483/d57d102e-a0f4-11e6-849b-0d997148bc0a.png)
 
```

Comment:
In bytes or screen real estate? It is intended to be 200x200, which seemed similar to some others I saw. I'm happy to try 150x or 100x. Open on that.